### PR TITLE
Test against Docker 1.7.1 RC2

### PR DESCRIPTION
**Pull request by aanand**

### CODE REVIEW

**Comment by dnephin** [comment]
LGTM

File: Dockerfile [hunk 1]
```diff
@@ -48,13 +48,13 @@ RUN set -ex; \
     rm -rf pip-7.0.1; \
     rm pip-7.0.1.tar.gz
 
-ENV ALL_DOCKER_VERSIONS 1.6.0 1.7.0
+ENV ALL_DOCKER_VERSIONS 1.6.0 1.7.1-rc2
```
Comment (thaJeztah) [inline Dockerfile]: off-topic for this PR, but any reason this still uses 1.6.0, not 1.6.2?

**Comment by thaJeztah** [comment]
@aanand interesting name for your branch btw :wink:

**Comment by thaJeztah** [comment]
And...history is repeating itself.. RC3 was just put out.. https://github.com/docker/docker/pull/14264#issuecomment-120132694

**Comment by aanand** [comment]
@thaJeztah Naming things is hard. And since RC3 is now out, I've amended the commit to ensure that both the version number and the release candidate number are incorrect.

**Comment by aanand** [comment]
Closing in favour of https://github.com/docker/compose/pull/1698.

**Pull request closed without merging**